src/popup/router/components/AccountInfo.vue

```diff
@@ -1,32 +1,39 @@
 <template>
-  <div :class="['account-info', { edit, copied }]">
-    <div class="account-buttons">
-      <ButtonPlain
-        v-if="idx !== 0 && $route.path === '/accounts'"
-        class="remove"
-        @click="remove"
-      >
-        <Remove />
-      </ButtonPlain>
-      <RouterLink
-        v-if="UNFINISHED_FEATURES && $route.path !== '/accounts'"
-        to="/accounts"
-      >
-        <Settings />
-      </RouterLink>
-      <ButtonPlain
-        v-if="UNFINISHED_FEATURES && idx === 0"
-        @click="createAccount"
-      >
-        <Add />
-      </ButtonPlain>
-      <ButtonPlain
-        v-clipboard:copy="accounts[idx].address"
-        data-cy="copy"
-        @click="copy"
-      >
-        <Copy />
-      </ButtonPlain>
+  <div :class="['account-info', { edit }]">
+    <div class="buttons">
+      <Component
+        :is="cardMinified ? 'Expand' : 'Collapse'"
+        class="minify"
+        @click="$store.commit('toggleMinifiedCard')"
+      />
+      <div>
+        <ButtonPlain
+          v-clipboard:copy="accounts[idx].address"
+          data-cy="copy"
+          @click="copy"
+        >
+          <Copy />
+        </ButtonPlain>
+        <ButtonPlain
+          v-if="UNFINISHED_FEATURES && idx === 0"
```

Comment:
```suggestion
          v-if="UNFINISHED_FEATURES"
```
Seems that `createAccount` button should be visible in each card. Can you elaborate on this @paolomolo?